config/security.java.policy-server

```diff
@@ -21,6 +21,8 @@ grant principal org.ow2.proactive.authentication.principals.GroupNamePrincipal "
     permission org.ow2.proactive.permissions.MethodCallPermission "org.ow2.proactive.resourcemanager.core.RMCore.releaseNode";
     permission org.ow2.proactive.permissions.MethodCallPermission "org.ow2.proactive.resourcemanager.core.RMCore.releaseNodes";
     permission org.ow2.proactive.permissions.MethodCallPermission "org.ow2.proactive.resourcemanager.core.RMCore.releaseBusyNodesNotInList";
+    permission org.ow2.proactive.permissions.MethodCallPermission "org.ow2.proactive.resourcemanager.core.RMCore.areNodesKnown";
```

Comment:
do we need to add releaseBusyNodesNotInList as well ?

Reply:
Yes `releaseBusyNodesNotInList` is still needed. We cannot somehow use `areNodesKnown` to replace `releaseBusyNodesNotInList`, if it was what you meant.